k8s/app-secret.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Secret
+metadata:
+    name: app-secret
+type: Opaque
+data:
+    postgres-password: dG9wc2VjdXJl
```

Comment:
Is there an alternative to having a hard-coded secret here? Does this file truly belong in source control?

Reply:
This is the equivalent of what we currently do in Docker Compose when setting the postgres password in the .env file, this is a minimal security configuration and will not actually be a secret, just a base64 encrypted script (in fact some blogs mention that the name secret is misleading) 

https://kubernetes.io/docs/concepts/configuration/secret/ shows a warning about "thinking" that the secrets are actually a security measure.

I think that for the purpose of the demo we could keep the file, but add a notice in the readme to use a different security method for passwords